tests/cephfs/multifs/multifs_kernelmounts.py

```diff
@@ -67,6 +67,20 @@ def run(ceph_cluster, **kw):
             extra_params=f",fs={fs_names[1]}",
             fstab=True,
         )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 400 "
+            f"--files 100 --files-per-dir 10 --dirs-per-dir 1 --top "
+            f"{kernel_mounting_dir_1}",
+            long_running=True,
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create --threads 10 --file-size 400 "
+            f"--files 100 --files-per-dir 10 --dirs-per-dir 1 --top "
+            f"{kernel_mounting_dir_2}",
+            long_running=True,
+        )
```

Comment:
Could we also run some IO's after reboot and cleanup cephfs data & mounts

Reply:
Updated the code to add IOs
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Z2ABZ5/